R/helpers_rweka.R

```diff
@@ -32,6 +32,12 @@ rweka_train = function(data, formula, pars, weka_learner, nested_pars = NULL) {
       names(ctrl[[par]]) = c("", gsub(suffix, replacement = "", x = arg_names_extra))
     }
   }
+  # add I or F params to control if there is weight param in IBk
+  if (grepl("IBk", capture.output(weka_learner)[1])) {
```

Comment:
what is the `grepl()` needed for?

Reply:
I want to make sure that we're only doing additional processing for `IBk` learner. RWeka doesn't show the name of learner in class but it does when you simply call the learner so I'm capturing this output and checking if it has `IBk` in its name.

Reply:
Ok! :)